packages/makeswift/src/errors/MakeswiftError.ts

```diff
@@ -3,7 +3,7 @@
  * where we want the program to simply exit.
  */
 export default class MakeswiftError extends Error {
-  constructor(message: string) {
+  constructor(message: string | undefined = '') {
```

Comment:
When a default value is used, it's not necessary to add `| undefined` as an argument. Also, when you want to make an argument optional, you can do `message?: string` instead of using `undefined` explicitly

Reply:
Cool, good feedback